apps/ex_nvr_web/test/ex_nvr_web/controllers/api/events_controller_test.exs

```diff
@@ -0,0 +1,157 @@
+defmodule ExNVRWeb.Api.EventsControllerTest do
+  @moduledoc false
+
+  use ExNVRWeb.ConnCase
+
+  import ExNVR.{AccountsFixtures, DevicesFixtures}
+
+  alias ExNVR.Events
+
+  @lpr_plate "01-D-12345"
+  @lpr_direction "forward"
+  @lpr_time ~U"2023-12-12T10:00:00Z"
+  @lpr_list_type "visitor"
+  @lpr_confidence "0.7"
+  @lpr_vehicle_type "Bus"
+  @lpr_vehicle_color "red"
+  @lpr_plate_color "white"
+  @lpr_coordinates_x1 "0"
+  @lpr_coordinates_y1 "100"
+  @lpr_coordinates_x2 "0"
+  @lpr_coordinates_y2 "50"
+  @lpr_plate_image "../fixtures/images/license-plate.jpg"
+                   |> Path.expand(__DIR__)
+                   |> Base.encode64()
+  @lpr_full_image "../fixtures/images/license-plate.jpg"
+                  |> Path.expand(__DIR__)
+                  |> Base.encode64()
+
+  setup %{conn: conn} do
+    device = device_fixture(%{vendor: "milesight", settings: %{storage_address: "/tmp"}})
+
+    token_conn = log_in_user_with_access_token(conn, user_fixture())
+    user_conn = log_in_user_with_username_password(
+      conn,
+      user_fixture(%{password: "Password1"}),
+      "Password1"
+    )
+
+
+    %{device: device, user_conn: user_conn, token_conn: token_conn}
+  end
+
+  describe "POST /api/events" do
+    test "create a new lpr event", %{user_conn: conn, device: device} do
+      conn
+      |> post(
+        ~p"/api/events?type=lpr&device_id=#{device.id}",
+        %{
```

Comment:
You should create an `EventFixture` module to get this attributes from like we did for the other entities.